src/System.IO.FileSystem/tests/WinRT_BrokeredFunctions.cs

```diff
@@ -0,0 +1,207 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Xunit;
+using Windows.Storage;
+
+namespace System.IO.Tests
+{
+    [PlatformSpecific(TestPlatforms.Windows)]
+    [SkipOnTargetFramework(~(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot))]
+    public partial class WinRT_BrokeredFunctions : FileSystemTest
+    {
+        private static string s_musicFolder = StorageLibrary.GetLibraryAsync(KnownLibraryId.Music).AsTask().Result.SaveFolder.Path;
+
+        [Fact]
+        public void CopyFile_ToBrokeredLocation()
+        {
+            string testFile = GetTestFilePath();
+            File.Create(testFile).Dispose();
+
+            string destination = Path.Combine(s_musicFolder, "CopyToBrokeredLocation_" + Path.GetRandomFileName());
+            try
+            {
+                Assert.False(File.Exists(destination), "destination shouldn't exist before copying");
+                File.Copy(testFile, destination);
```

Comment:
My only concern here is the tight timeframe we have for these tests to run. I'll move them to OuterLoop after this moves to rel. (As this only runs for UAP it is pretty unlikely to hit anyone in the short term.)